<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dict</class>
 <widget class="QWidget" name="Dict">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1883</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>简易词典</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_en2cn">
       <property name="text">
        <string>英-&gt;中</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_en2cn">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>hello</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_cn2en">
       <property name="text">
        <string>中-&gt;英</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_cn2en">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>你好</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>查词结果</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
